src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/reports/hearingsbyhearingtype/HearingsByHearingTypeReport.java

```diff
@@ -108,27 +108,34 @@ private void initReportSummary2HdrList(List<HearingsByHearingTypeReportSummary2H
 
     }
 
-    private String getSubSplit(HearingTypeItem hearingTypeItem) {
+    private String getSubSplitSitAlone(HearingTypeItem hearingTypeItem) {
         if ("Full".equals(hearingTypeItem.getValue().getHearingSitAlone())) {
```

Comment:
Is the comparison null-safe if hearingTypeItem.getValue() returns null?

Reply:
refactored